tests/test_crash_reports.py

```diff
@@ -107,6 +107,39 @@ def test_that_top_crashers_reports_links_work(self, mozwebqa, product):
             top_crashers = csp.release_channels
 
     @pytest.mark.nondestructive
+    def test_top_crasher_reports_tab_has_uuid_report(self, mozwebqa):
```

Comment:
Thanks @bobsilverberg -- wow, the *new* test caught a regression that didn't exist earlier this week. Momentarily, I'll have an amended pr in a moment with an xfail added.